models/demos/t3000/falcon40b/tt/falcon_decoder.py

```diff
@@ -217,7 +217,7 @@ def fwd_prefill(
         )
 
         if self.model_config["LN_INPUT_DTYPE"] != self.model_config["BFP8_DTYPE"]:
-            replicated_hidden_states = ttnn.experimental.typecast(
```

Comment:
this can to stay ttnn.experimental.typecast. it refers to typecast in ttnn/operations/experimental/copy